tmt/cli.py

```diff
@@ -174,9 +174,15 @@ def main(context, root, **kwargs):
     '-i', '--id', 'id_', help='Run id (name or directory path).', metavar="ID")
 @click.option(
     '-a', '--all', 'all_', help='Run all steps, customize some.', is_flag=True)
+@click.option(
+    '-e', '--env', 'env_',
```

Comment:
Can we use `--environment` for the long option? I think we can add option shortening (similar as we already have for commands) directly into click if needed/desired.

Reply:
Doesn't click already auto-support shortening?
(IOW if there's only one option that begings with ....)

Reply:
No, it does not:

    tmt plan --hel
    Usage: tmt plan [OPTIONS] COMMAND [ARGS]...
    Try "tmt plan --help" for help.

    Error: no such option: --hel  Did you mean --help?

For command shortening I needed to do [this](https://github.com/psss/tmt/blob/master/tmt/cli.py#L29).